src/js/juju-cards-v1.0.0.js

```diff
@@ -116,6 +122,17 @@ let jujuCards = () => {
       detailsLink = `${ownerLink}/${name}`;
     }
     let addLink = `${demoDomain}/?deploy-target=${id}`;
+    let deployTitle = 'Deploy with Juju';
+
+    if (typeof(card.dataset.dd) !== 'undefined') {
```

Comment:
This `addLink`/`deployTitle` toggle is now here twice, can you abstract it out to it's own method?